makefile

```diff
@@ -174,7 +174,7 @@ vision-torch-tf-setup: env
 
 nlp-tests-setup: env
 	@echo "####  installing nlp properties packages #### "
-	$(PIP) install -q "langdetect>=1.0.9" "textblob>=0.17.1";
+	$(PIP) install -q "langdetect>=1.0.9" "textblob>=0.17.1" "fasttext>=0.8.0"
```

Comment:
I think I've added these to nlp-properties. Do we need this individual setup or can we use the nlp-properties req file?

Reply:
we can, but the `nlp-prop-requirements.txt` file does not contain this one - `langdetect>=1.0.9`,
should I add it, or it is unneeded? As I see it is not used anymore?

Reply:
We don't need langdetect anymore, just fasttext

Reply:
fixed